
import * as LANGUAGE from '@app/constants/language'

export default {
  [LANGUAGE.ENGLISH]: '英文',
  [LANGUAGE.CHINESE]: '简体中文',

  [LANGUAGE.HOME]: '明殿',
  [LANGUAGE.GUESTBOOK]: '百鸣苑',
  [LANGUAGE.ABOUT]: '狂浪生',
  
  [LANGUAGE.ORIGIN_HYBRID]: '混撰',
  [LANGUAGE.ORIGIN_ORIGINAL]: '原创',
  [LANGUAGE.ORIGIN_REPRINT]: '转载',
  [LANGUAGE.AM]: '上午',
  [LANGUAGE.PM]: '下午',
  
  [LANGUAGE.ARTICLE]: '文章',
  [LANGUAGE.TAG]: '标签',
  [LANGUAGE.COMMENT]: '评论',
  [LANGUAGE.TODAY_VIEWS]: '今日阅读',

  [LANGUAGE.GITHUB]: '同性交友',
  [LANGUAGE.VLOG]: '浪迹天涯',
  [LANGUAGE.EMAIL_ME]: '给我邮件',

  [LANGUAGE.TWITTER]: '推特',
  [LANGUAGE.LINKEDIN]: '领英',
  [LANGUAGE.WEIBO]: '微博',

  [LANGUAGE.SETTING]: '设置',
  [LANGUAGE.CLEAR_CACHE]: '清理缓存',
  [LANGUAGE.SWITCH_LANGUAGE]: '切换语言',
  [LANGUAGE.DARK_THEME]: '夜间模式',

  [LANGUAGE.CLEAR_CACHE_TEXT]: '这将清除所有的本地数据，继续吗？',
  [LANGUAGE.CLEAR_CACHE_OK_BUTTON]: '确认',
  [LANGUAGE.CLEAR_CACHE_CANCEL_BUTTON]: '取消',

  [LANGUAGE.SUCCESS]: '操作成功',
  [LANGUAGE.NETWORK_ERROR]: '网络请求错误',
  [LANGUAGE.UNKNOW_ERROR]: '未知错误',
  [LANGUAGE.CALL_EMAIL_ERROR]: '不知道啥原因，反正调不起邮件客户端',
}